Containers/mastercontainer/cron.sh

```diff
@@ -1,12 +1,36 @@
-#!/bin/sh
-set -eux
+#!/bin/bash
 
 while true; do
+    if [ -f "/mnt/docker-aio-config/data/daily_backup_time" ]; then
+        set -x
+        BACKUP_TIME="$(cat "/mnt/docker-aio-config/data/daily_backup_time")"
+        DAILY_BACKUP=1
+        set +x
+    else
+        BACKUP_TIME="04:00"
+        DAILY_BACKUP=0
+    fi
+
+    while [ $(date +%H:%M) != "$BACKUP_TIME" ]; do 
+        sleep 1
+    done
+
+    if [ "$DAILY_BACKUP" = 1 ]; then
+        # TODO: below
+        # 1. delete all sessions and block login
+        # 2. After 5min waiting, Update the mastercontainer (and check if watchtower is running)
```

Comment:
maybe also see if one of the longer running things started: e.g backup container.
also better wait 10mins